pulsar-client/src/main/java/org/apache/pulsar/client/impl/PatternMultiTopicsConsumerImpl.java

```diff
@@ -203,9 +205,14 @@ public CompletableFuture<Void> onTopicsAdded(Collection<String> addedTopics) {
                 return addFuture;
             }
 
+            Set<String> addTopicPartitionedName = addedTopics.stream()
```

Comment:
Do you need to fix the https://github.com/apache/pulsar/pull/16719/files#diff-779a47f80b0048bab183870ed4c1ecd07ffd538006cf9358e494bee23405d924R179?

There has the same logic with this.

Reply:
Do you mean removeTopic? there is no problem with the implementation of removeTopic.

Reply:
Okay.